Specs/Scene/ImplicitTileCoordinatesSpec.js

```diff
@@ -209,7 +1501,8 @@ describe("Scene/ImplicitTileCoordinates", function () {
   it("computes the morton index for octree coordinates", function () {
     var coordinates = new ImplicitTileCoordinates({
       subdivisionScheme: ImplicitSubdivisionScheme.OCTREE,
-      level: 4,
+      subtreeLevels: 8,
+      level: 6,
```

Comment:
This test used to be invalid because it was level 4 and had a z value of 32. Had to raise the level to fit.